portfolio/src/main/webapp/script.js

```diff
@@ -79,16 +174,26 @@ function createFootprints(locationID) {
         footprint.src = "images/footprint.png";
         const direction = (i % 2 == 0) ? "right" : "left";
         footprint.classList.add("footprint-" + direction);
-
-        /* Vary width between steps for a more natural appearance. */
-        const rand = Math.floor(Math.random * (3) + 2);
+        const rand = Math.floor(Math.random * (4) + 3);
         if (direction.localeCompare("right") == 0) {
             footprint.style.margin = "1rem 1rem 1rem " + rand + "rem";
         } else {
             footprint.style.margin = "1rem " + rand + "rem 1rem 1rem";
         }
-        
         footprint.classList.add("fade");
         location.appendChild(footprint);
     }
 }
+
+/* 
+ * fsdf   sdsdfsent using ID as identifier. Does not redirect page after
```

Comment:
?!?

Reply:
!!! So this is where my keyboard was at when I thought the Cloud Console had froze...